docs/SUMMARY.md

```diff
@@ -91,21 +91,27 @@
   * [Code of Conduct](contributing-to-airbyte/code-of-conduct.md)
   * [Developing Locally](contributing-to-airbyte/developing-locally.md)
   * [Developing Connectors](contributing-to-airbyte/building-new-connector/README.md)
+    * [Using the Airbyte CDK](contributing-to-airbyte/using-the-airbyte-cdk/README.md)
```

Comment:
For this README.md why don't we add something like 

```
Airbyte Connector Development Kits (CDK) which significantly accelerate connector development. We currently provide a Python CDK, and a Java one is in development. 

Navigate here for more information on the [Python CDK](link)
```

Reply:
Good idea, done.